auth-tokens-filter/src/main/java/com/palantir/tokens/auth/http/BasicAuthToBearerTokenFilter.java

```diff
@@ -115,7 +114,7 @@ private AuthHeader base64DecodePassword(String rawAuthHeader) {
                 rawAuthHeader.substring(BASIC_AUTH_STR.length()).trim();
         String credentials;
         try {
-            credentials = new String(BASE_64_ENCODING.decode(base64Credentials), StandardCharsets.UTF_8);
```

Comment:
static field was unnecessary because the getUrlDecoder method returns a static field -- indirection of a method invocation is insignificant.